ckanext/qdes_schema/plugin.py

```diff
@@ -28,8 +34,17 @@ class QDESSchemaPlugin(plugins.SingletonPlugin):
     plugins.implements(plugins.IFacets, inherit=True)
     plugins.implements(plugins.IAuthFunctions)
 
-    # IBlueprint
 
+    # IConfigurable
+    def configure(self, config):
+        core_validator.object_id_validators['publish external schema'] = core_validator.package_id_exists
```

Comment:
@awset there's a number of label/comment references in here to "external schema" - some are a bit confusing, e.g. the activity name 'publish external schema' ... what we're actually doing is publishing to an external catalogue - the schema is implemented by the catalogue, but to a user we're just publishing to an external catalogue

I'm not suggesting to go change all references of "external schema" to "external catalogue" - just where it makes sense, i.e. here in the activity name would refer to catalogue, and I guess if following the CKAN standard naming convention for activities, e.g. `'deleted group'` we'd name it `published external catalogue`